avm/res/machine-learning-services/registry/version.json

```
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
+  "version": "0.9",
+  "pathFilters": [
+    "./main.json"
+  ]
```

```suggestion
  "version": "0.1"
```

Should do the trick 😉